tests/materials-page/materials-page.query.test.js

```diff
@@ -0,0 +1,11 @@
+const { setupApp } = require('../helper-functions');
+const { getAllMaterialsBlocks } = require('./materials-page.helper');
+
+describe('Materials-about queries', () => {
+  it('Should get all materials-blocks', async () => {
+    const operations = await setupApp();
+    const blocks = await getAllMaterialsBlocks(operations);
+    expect(blocks).toBeDefined();
+    expect(blocks.items).toBeInstanceOf(Array);
```

Comment:
I suspect that "blocks" will always be defined no matter there is an error or successful response.

Reply:
fix)